templates/web/submits_show_source.html

```diff
@@ -20,7 +20,12 @@
     </div>
     <div id="submit-{{ submit.assignment.id }}-{{submit.student}}" class="collapse">
       <div class="card-body">
-        <div class="comments"><kelvin-submit-sources url="{% url 'submit_comments' submit.assignment.id submit.student submit.submit_num %}"></kelvin-submit-sources></div>
+        <div class="comments">
+            <kelvin-submit-sources
+                    url="{% url 'v2:retrieve_submit_details' submit.assignment.id submit.student submit.submit_num %}"
+                    commentUrl="{% url 'v2:create_submit_comment' submit.assignment.id submit.student submit.submit_num %}"
```

Comment:
```suggestion
                    comment_url="{% url 'v2:create_submit_comment' submit.assignment.id submit.student submit.submit_num %}"
```
Otherwise the prop ends up being `undefined`.